src/Payload/AbstractPayload.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\Payload;
+
+abstract class AbstractPayload implements PayloadInterface
```

Comment:
It's better not to have it. Especially such `getName()` implementation. It leads to coupling with the class name and thus making it harder to consume message in another service.